Makefile

```diff
@@ -1,3 +1,7 @@
+export PYTHONUNBUFFERED = 1
+export MOZ_HEADLESS = 1
+export GECKODRIVER_LOG = $(shell pwd)/results/geckodriver.log
```

Comment:
If we're exporting these we should capture them as an artifact, it could help with future debugging.